src/lib/oauth2-server.ts

```diff
@@ -19,6 +19,8 @@
  * @module lib/oauth2-server
  */
 
+import * as fs from 'fs';
```

Comment:
Why not just `import { readFileSync } from 'fs'`?

Reply:
Good call, fixed!